build.gradle

```diff
@@ -18,10 +18,11 @@ allprojects {
         options.fork = true  // Fork your compilation into a child process
         options.forkOptions.setMemoryMaximumSize("4g") // Set maximum memory to 4g
         options.compilerArgs << "-Xlint:all"
-        options.compilerArgs << "-Werror"
+        // options.compilerArgs << "-Werror"
```

Comment:
@pfent is there not some config file to add exceptions for those libs? I remember seeing it _somewhere_ but not exactly...